website/blog/2018-07-27-removing-babels-stage-presets.md

```diff
@@ -0,0 +1,193 @@
+---
+layout: post
+title:  "Removing Babel's Stage Presets"
+author: Henry Zhu
+authorURL: https://twitter.com/left_pad
+date:   2018-07-27 12:00:00
+categories: announcements
+share_text: "Removing Babel's Stage Presets"
+---
+
+Moving forward with v7, we've decided it's best to stop publishing the Stage presets in Babel (e.g. `@babel/preset-stage-0`).
+
+We didn't make this decision lightly and wanted to show the context behind the interplay between TC39, Babel, and the community.
+
+<!--truncate-->
+
+## Some History
+
+A Babel preset is a shareable list of plugins.
+
+The [official Babel Stage presets](https://babeljs.io/docs/en/next/presets) tracked the [TC39 Staging process](https://tc39.github.io/process-document/) for new [syntax proposals](https://github.com/tc39/proposals) in JavaScript.
```

Comment:
Shouldn't it be `ECMAScript` instead of `JavaScript` here? Does it matter?

Reply:
It's interchangeable for the most part, and maybe many might not of heard of it (similar to ES6 vs. ES2015)